examples/bri-3/src/bri/transactions/api/transactions.controller.spec.ts

```diff
@@ -108,8 +107,8 @@ describe('TransactionController', () => {
 
     it('should return the correct transaction if proper id passed ', async () => {
       // Arrange
-      const fromBpiSubjectAccount = await createBpiSubjectAccount('123');
-      const toBpiSubjectAccount = await createBpiSubjectAccount('321');
+      const fromBpiSubjectAccount = createBpiSubjectAccount(uuid());
```

Comment:
why uuid() here and '123' in the createTransaction section bellow?

Reply:
probably because test is checking if uuid is well formed, will check

Reply:
is this still open?

Reply:
no,fixed i am ussing uuid() everywhere